packages/webapp/src/util/convert-units/unit.js

```diff
@@ -113,6 +114,61 @@ export const container_planting_depth = {
   databaseUnit: databaseUnit.length,
 };
 
+export const estimated_duration = {
+  metric: {
+    units: ['h', 'm'],
+    defaultUnit: 'h',
+    breakpoints: [9999999999],
+  },
+  imperial: {
+    units: ['h', 'm'],
+    defaultUnit: 'h',
+    breakpoints: [9999999999],
+  },
+  databaseUnit: databaseUnit.hour,
+};
+
+export const irrigation_depth = {
+  metric: {
+    units: ['mm', 'm'],
```

Comment:
Kevin mentioned that irrigation depth should never be m, you can remove m. This also relates to my comments on `<Unit />`